enhancements/cluster-logging/cluster-logging-log-forwarding.md

```diff
@@ -104,198 +156,160 @@ This is often required for industries such as the US public sector, healthcare o
 * A pipeline shall support at least one output.
 * A pipeline may support multiple outputs.
 * Outputs that require secrets which are not created by the cluster-logging-operator shall be created and managed by the administrator of the endpoint
-* Secrets shall have keys of: `tls.crt`, `tls.key`, `ca-bundler.crt` which point to the respective certificates for which they represent. Secrets shall have the key `shared_key` for use when using `forward` in a secure manner
-* The operator will only support a single instance of log forwarding named `instance` for at least the tech preview release
-* Log forwarding is enabled for tech preview when the instance of `clusterloggging` is annotated with: `clusterlogging.openshift.io/logforwardingtechpreview: enabled`
+* Forwarder CRs can refer to secrets for TLS certificates (see security section below)
+* The operator allows multiple instances of `forwarder`.
+  * Instances in the `openshift-logging` namespace are "global" and can forward logs from any container.
+  * Instances in other namespaces are namespace-restricted and can only forward logs from their own namespace.
```

Comment:
is there an implication here that namespace forwarders can only forward application logs, not "infra" and "audit" logs?

if so this may answer my earlier questions.

Reply:
Correct.